noxfile.py

```diff
@@ -131,7 +131,8 @@ def clippy(session: nox.Session) -> bool:
 def _clippy(session: nox.Session, *, env: Dict[str, str] = None) -> bool:
     success = True
     env = env or os.environ
-    for feature_set in _get_feature_sets():
+    jiff_features_set = (("--features=jiff-01",), ("--features=jiff-01 abi3",))
+    for feature_set in _get_feature_sets() + jiff_features_set:
```

Comment:
The clippy jobs are already quite slow. Rather than adding additional jobs here, please can you modify `_get_feature_sets()` to be rust version aware and only set the jiff feature if the compiler is new enough?